spec/controllers/reservations_controller_spec.rb

```diff
@@ -29,8 +30,45 @@
     end
   end
 
+  describe "DELETE #destroy" do
+    context "when user is logged in" do
+      before do
+        sign_in_user
+        delete_destroy(wishlist.id, item.id, reservation.id)
```

Comment:
You think it works this way, or you've checked and you're sure it works this way? [The docs](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#associations) say nothing about such assumptions made by the FactoryGirl.